pyanaconda/ui/webui/__init__.py

```diff
@@ -90,25 +94,59 @@ def run(self):
         """Run the interface."""
         log.debug("web-ui: starting cockpit web view")
 
-        # Force Firefox to be used via the BROWSER environment variable.
-        # This is read by cockpit-desktop and makes it launch Firefox in kiosk mode
-        # instead of the GTK WebKit based web view it launches by default.
-
-        # FIXME: looks like "type" should not be used and _is_live_os is private ?
         if conf.system.provides_liveuser:
-            profile_name = FIREFOX_THEME_LIVE
+            self._watch_webui_on_live()
         else:
-            profile_name = FIREFOX_THEME_DEFAULT
+            self._run_webui()
+
+    def _run_webui(self):
+        # FIXME: This part should be start event loop (could use the WatchProcesses class)
+        # FIXME: We probably want to move this to early execution similar to what we have on live
+        profile_name = FIREFOX_THEME_DEFAULT
```

Comment:
The live vs boot.iso need different firefox theming. This is not a desired change.

Reply:
It is just code move. This code should be started only on boot.iso not on live so the theme here is correct. The other theme is specified in `liveinst` script because there is a place where the script is started.